src/Microsoft.AspNetCore.OData/Microsoft.AspNetCore.OData.csproj

```diff
@@ -21,6 +21,11 @@
     <PackageReference Include="Newtonsoft.Json" Version="10.0.3" />
   </ItemGroup>
 
+  <!-- Should be done for .NET 5 and above as well. Then usage of NetCore3 and above should be fixed to support Net5 as well.-->
+  <PropertyGroup Condition="'$(TargetFramework)' == 'netcoreapp3.1' ">
+    <DefineConstants>$(DefineConstants);NETCORE3</DefineConstants>
```

Comment:
>NETCORE3 [](start = 40, length = 8)

Needn't to define a new constant, there's a "NETCOREAPP3_1" predefined.

Reply:
Done. Added for Net5 as well.